demos/common/python/pipelines/sync_pipeline.py

```diff
@@ -0,0 +1,40 @@
+"""
+ Copyright (C) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+
+class SyncPipeline:
+    def __init__(self, ie, model=None, device='CPU', silent=False):
+        self.logger = logging.getLogger() if not silent else lambda *x: None
+
+        self.ie = ie
+        self.device = device
+        if model:
+            self.model = model
+            self.logger.info('Loading network to {} plugin...'.format(device))
+            self.exec_net = ie.load_network(network=self.model.net, device_name=device)
+
+    def reload_model(self, model, device=None):
+        if device:
+            self.device = device
+        self.model = model
+        self.logger.info('Loading network to {} plugin...'.format(device))
```

Comment:
It prints ` Loading network to None plugin...`, which is strange.

Regarding overall reports of net loading, do we really have to have that spam of prints?

Reply:
Done

Reply:
> Regarding overall reports of net loading, do we really have to have that spam of prints?
Well, it is more legacy from samples. So I think we could remove it in future